plugin.rb

```diff
@@ -47,6 +47,8 @@ class Engine < ::Rails::Engine
   require_relative "app/lib/web_hook_extension"
   require_relative "app/serializers/concerns/topic_answer_mixin"
 
+  require_relative "app/lib/plugin_initializer"
+  DiscourseSolved::AssignsReminderForTopicsQuery.new(self).apply_plugin_api
```

Comment:
Is there a better way to do this?

Reply:
When there is only one... probably not. If there were multiple files you would loop over each at require, apply each one. This is probably fine. Although the name of the file should be more specific. Like `app/lib/plugin_initializer.rb` -> `lib/plugin_initializers/assigned_reminder_exclude_solved.rb`